README.md

```diff
@@ -78,5 +77,17 @@ If you find this work and/or code useful for your research, please cite our pape
 	journal = {medRxiv},
 	note = {*Shared first authorship}
 }
+```
 
 ```
+@article {karthik2024scisegv2universaltoolsegmentation,
+      title={SCIsegV2: A Universal Tool for Segmentation of Intramedullary Lesions in Spinal Cord Injury}, 
+      author={Enamundram Naga Karthik* and Jan Valošek* and Lynn Farner and Dario Pfyffer and Simon Schading-Sassenhausen and Anna Lebret and Gergely David and Andrew C. Smith and Kenneth A. Weber II au2 and Maryam Seif and RHSCIR Network Imaging Group and Patrick Freund and Julien Cohen-Adad},
+      year={2024},
+      eprint={2407.17265},
+      archivePrefix={arXiv},
+      primaryClass={cs.CV},
+      url={https://arxiv.org/abs/2407.17265}, 
```

Comment:
I got this ref from [arXiv](https://arxiv.org/abs/2407.17265) by clicking on "Export BibTeX Citation" button. But the fields are now different from SCIsegV1 ref. Maybe we could unify them, right?

Reply:
I think it's okay, don't know what to remove, it's a copy-able block so it should be fine